src/main/java/core/basesyntax/SqrtOperation.java

```diff
@@ -8,6 +8,15 @@
  */
 public class SqrtOperation {
     public double sqrt(double number) {
-        return 0.0;
+        double sqrtNumber = 0;
+        try {
```

Comment:
why do we need this try catch block? 
Math.sqrt throws IllegalArgumentException ?